readme.md

```diff
@@ -14,32 +20,32 @@
 
 # Getting started
 
-At this moment development is in early stages but mostly stable. 
-
-You will need to look around existing projects and examples folder until documentation is written and the project a little more polished
+**NOTE:** Hivemind-core is also available as a Mycroft skill. You can use it *either* as a standalone program *or* as a Mycroft skill, but you can't run them both at the same time. During early development, testers are encouraged to choose this repository over the skill, as development takes place here.
 
-The easiest way to get started is by using the [Hivemind Skill](https://github.com/JarbasSkills/skill-hivemind)
+---
 
-The main configuration can be found at
+At this moment development is in early stages, but mostly stable. Functionality is limited to basic questions and answers, but only because we haven't implemented the bit that lets Mycroft re-activate the mic to continue a discussion. 
 
-    '~/.cache/json_database/HivemindCore.json'
+Full tutorials will follow later. For now, you wanna do this:
 
-# Components
+1. Clone this repo onto the computer or device that's running Mycroft.
+2. Note the scripts in the `examples` folder. These are enough to spin up your Hive and connect to Mycroft from another device.
+3. Edit `examples/add_keys.py` to create names and keys for your devices. You can use HiveMind without encryption, but it's discouraged for security reasons.
+4. Ensure Mycroft is running.
+5. Run `examples/mycroft_master.py`. Your Hive is now running, and you're ready to connect. See below to find a client (ideally the Voice Satellite). You will need to know the hostname or IP address of the computer or device running Mycroft, and you'll also need the device name and key you created earlier.
```

Comment:
theres auto discovery, no need to know ip address, maybe also mention that? i know you had trouble with it, but i never had and @j1nx and some other report it works fine